translate/snippets/client.go

```diff
@@ -12,33 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package trsnip contains snippet code for the Translate API.
-// The code is not runnable.
-package trsnip
+package snippets
```

Comment:
Is this sample used anywhere? If not, let's delete it. If so, might want to rename the file to something more specific than `client`.

Reply:
Done